man/adoc/bpftrace.adoc

```diff
@@ -45,6 +45,30 @@ List all the probes attached in the program::
 # bpftrace -l -e 'kprobe:do_nanosleep { printf("%d sleeping\n", pid); }'
 ----
 
+Run bpftrace in the background using systemd::
```

Comment:
I think it'd be better to add this to `Advanced Topics` section. Might confuse folks so early up in man page - mostly cuz it's still a default-off feature